website_snippet_product_category/views/product_public_category.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <data>
+        <record id="product_public_category_form_view" model="ir.ui.view">
+            <field name="name">public.category.form</field>
+            <field name="model">product.public.category</field>
+            <field
+                name="inherit_id"
+                ref="website_sale.product_public_category_form_view"
+            />
+            <field name="arch" type="xml">
+                <field name="sequence" position="after">
+                    <field name="website_published" />
+                    <field
+                        name="website_image"
+                        widget="image"
+                        options="{'size': [32,32]}"
```

Comment:
Is this limiting the uploaded image to 32x32?

Reply:
Changed, now uses existing "image" fields (image, image_medium, image_small) that resizes the image automatically.